src/confluent_kafka/admin/_config.py

```diff
@@ -177,3 +204,28 @@ def set_config(self, name, value, overwrite=True):
         if not overwrite and name in self.set_config_dict:
             return
         self.set_config_dict[name] = value
+
+    def set_incremental_config(self, name, operation, value=None):
+        """
+        Incrementally updates a configuration value.
+
+        Applies an incremental operation on specified key, while keeping
+        all the others unchanged.
+
+        :param str name: Configuration property name
+        :param AlterConfigOpType operation: Alter config operation
+        :param str value: Configuration value (optional if operation is DELETE)
+        """
+        if name is None:
+            raise TypeError("Configuration name is needed")
+        if not isinstance(name, str):
+            raise TypeError("Configuration name must be a string")
+
+        if not isinstance(operation, AlterConfigOpType):
+            raise TypeError("Operation must be of type AlterConfigOpType")
+
+        if not isinstance(value, str) and operation != AlterConfigOpType.DELETE:
+            raise ValueError("The provided value should be a string for: " + operation.name)
+        if value is None and operation != AlterConfigOpType.DELETE:
+            raise ValueError("Value is needed for operation: " + operation.name)
+        self.incremental_config[name] = {"operation_type": operation, "value": value}
```

Comment:
Why are we keeping this as `dict` instead of `list of tuples`? like `[(k1, o1, v1), ...]` instead of `{k1: {operation_type: o1, value: v1}, ...}`?

Reply:
It can be overwritten too

Reply:
Now that we are allowing multiple values for a config, does it make sense to take input as the above mentioned way? or in for of `list(ConfigEntry)` like we do in librdkafka? Just a thought.